web/src/app/core/resolvers/view-project.resolver.ts

```diff
@@ -13,18 +16,20 @@ import { IProject, ProjectModel } from '@shared/models/index.model';
 export class ViewProjectResolver implements Resolve<IProject> {
 
   constructor(
+    private fns: AngularFireFunctions,
     private projectService: ProjectService,
     private router: Router
   ) { }
 
   resolve(route: ActivatedRouteSnapshot): Observable<IProject> {
+    const callable: any = this.fns.httpsCallable('updateProjectViews');
     return this.projectService.findOneById(route.params.projectUid)
       .pipe(
         take(1),
-        switchMap((project: ProjectModel) => this.projectService.incrementView(project)),
+        tap(() => callable({ projectUid: route.params.projectUid })),
```

Comment:
The previous service method `incrementView` is no longer needed as it has been moved to the cloud function.

Reply:
I will remove this and push it up